Automattic/jetpack
# [Plugin] Backup: Implement ConnectionScreenRequiredPlan and PricingCard

#### Changes proposed in this Pull Request:

* Implement ConnectionScreenRequiredPlan and PricingCard

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Ensure the New screens appear
* Login link and Get Jetpack Backups should connect Jetpack (Will not end up in cart until wp-calypso flow changes)
* Get Jetpack Backups from the No Plan screen should go straight to cart

New Connect Screen
<img width="1536" alt="Screen Shot 2021-12-15 at 11 25 19 AM" src="https://user-images.githubusercontent.com/1992658/146224928-82025e0f-8ce8-4e31-a787-5880a4a86e49.png">
* Log in alignment fix is in another PR here: https://github.com/Automattic/jetpack/pull/22107

New "No Plan" Screen
<img width="1474" alt="Screen Shot 2021-12-15 at 11 24 40 AM" src="https://user-images.githubusercontent.com/1992658/146224820-7a095f14-2fd6-4174-8a4d-874c4096376f.png">
